float64[7] q_error
float64[7] q_commanded
float64[7] q_measured
float64 root_mean_square_error
